scripts/check-proof-systems-submodule.sh

```diff
@@ -10,11 +10,20 @@ git config http.sslVerify false
 git fetch origin
 git config http.sslVerify true
 
+if [ "${BUILDKITE_PULL_REQUEST_BASE_BRANCH}" = "" ]; then
+  echo "BUILDKITE_PULL_REQUEST_BASE_BRANCH env variable is not set. Are you triggered build manually?"
+  echo "If yes, then rerun it using custom build with environment variables:"
+  echo "BUILDKITE_PULL_REQUEST_BASE_BRANCH={your_target_branch}"
+  exit 1
+fi
+
 declare -A BRANCH_MAPPING=(
   ["rampup"]="compatible" 
   ["berkeley"]="berkeley"
   ["develop"]="develop"
   ["izmir"]="master"
+  # Default case
+  ["${BUILDKITE_PULL_REQUEST_BASE_BRANCH}"]="${BUILDKITE_PULL_REQUEST_BASE_BRANCH}"
```

Comment:
I don't think this is the desired default. We should probably default to `master`

Reply:
Defaulting would make this part fail when running the CI in a branch B targeting a branch different from master/develop/berkeley. This happened for example in the Pickles fixes PR train, where you took care of having matching branch names in proof-systems/snarkyjs.. So I kind of thought this wouldn't be bad discipline to enforce that.